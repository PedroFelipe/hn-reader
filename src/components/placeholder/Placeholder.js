import React, { Component } from 'react';

import 'bulma/css/bulma.css';
import './placeholder.css';

class Placeholder extends Component {
  render() {
    return (
      <div className="placeholder">
        <div className="placeholder-text">
          <span className="placeholder-background"></span>
        </div>

        <div className="placeholder-text">
          <span className="placeholder-background"></span>
        </div>
      </div>
    );
  }
}

export default Placeholder;
