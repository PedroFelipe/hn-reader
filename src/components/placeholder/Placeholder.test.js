import React from 'react'
import renderer from 'react-test-renderer';

import Placeholder from './Placeholder'

test('Placeholder renders correctly', () => {
  const tree = renderer.create(
    <Placeholder />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
