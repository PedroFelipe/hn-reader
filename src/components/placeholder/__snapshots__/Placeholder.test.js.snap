// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Placeholder renders correctly 1`] = `
<div
  className="placeholder"
>
  <div
    className="placeholder-text"
  >
    <span
      className="placeholder-background"
    />
  </div>
  <div
    className="placeholder-text"
  >
    <span
      className="placeholder-background"
    />
  </div>
</div>
`;
